packages/realm/src/Realm.ts

```diff
@@ -644,7 +682,11 @@ export class Realm {
     }
     this.internal.verifyOpen();
     const helpers = this.classes.getHelpers(type);
-    return RealmObject.create(this, values, mode, { helpers });
+    const realmObject = RealmObject.create(this, values, mode, { helpers });
```

Comment:
General Question: Does `Realm.Object.create` actually return something in asymmetric mode?  If so, are we taking up any memory when we call `create`?

Reply:
Core [returns `Object{}`](https://github.com/realm/realm-core/blob/792fd5bdde22716edaa4d7f3850cfcce3ee5642d/src/realm/object-store/object_accessor.hpp#L365-L367) and we do an [`assert(result)`](https://github.com/realm/realm-js/blob/8f395b1e8c0b943e604de964012dc80f21e773d7/packages/realm/src/Object.ts#L134) in the `RealmObject.create()` method. So I believe this would have thrown if an object was not returned.